.env.example

```
@@ -1,5 +1,11 @@
 API_PORT=3000
 
+JWT_AT_SECRET=jwt-secret
+JWT_AT_MAX_AGE_S=3600
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_AT_MAX_AGE_S key should go before the JWT_AT_SECRET key

(UnorderedKey)